tests/phpunit/api/v3/PaymentTest.php

```diff
@@ -403,4 +415,141 @@ public function testUpdatePayment() {
     ));
   }
 
+  /**
+   * Test create payment api for paylater contribution
+   */
+  public function testCreatePaymentPayLater() {
+    $this->createLoggedInUser();
+    $contributionParams = array(
+      'total_amount' => 100,
+      'currency' => 'USD',
+      'contact_id' => $this->_individualId,
+      'financial_type_id' => 1,
+      'contribution_status_id' => 2,
+      'is_pay_later' => 1,
+    );
+    $contribution = $this->callAPISuccess('Contribution', 'create', $contributionParams);
+    //Create partial payment
```

Comment:
This doesn't look like a partial payment - the amount is $100 and the contribution amount is $100